hack/shellcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then
+  find "${1:-.}" -type f -name '*.sh' -exec shellcheck --format=gcc {} \+
+else
+  docker run -e IS_CONTAINER='TRUE' --rm -v "$(pwd)":/workdir:ro --entrypoint sh quay.io/coreos/shellcheck-alpine:v0.5.0 /workdir/hack/shellcheck.sh /workdir;
```

Comment:
nit: I [think][1] this should use `"$(realpath "${DIR}")"` instead of `"$(pwd)"`.  That way callers could point the containerized check at a specific location if they didn't want to recurse through the whole current working directory.  Although with that approach you'd need to either mount this script somewhere else or copy the `find` command on this line to avoid the script dependency inside the container.  Anyway, just ideas for future polishing; nothing to block this PR on.

[1]: https://github.com/openshift/installer/pull/95#discussion_r207060596